docs/src/contributing.md

```diff
@@ -1,7 +1,7 @@
 # Contributing
 
 The BioJulia organisation has a set of contribution guidelines which apply to
-all BioJulia projects. These guidelines are available [here](http://biojulia.github.io/Contributing/latest) and it is
+all BioJulia projects. These guidelines are available [here](/CONTRIBUTING.md) and it is
```

Comment:
I think the link in this file is just linking to itself, no? I think we want to link to https://github.com/BioJulia/BiobakeryUtils.jl/blob/main/CONTRIBUTING.md

Reply:
Actually, perhaps this is a better source of truth: https://github.com/BioJulia/Contributing/blob/master/CONTRIBUTING.md

Reply:
Okay got it. I linked it to the file in the main repo, which I think is pretty similar? But I can change the link again.